src/AuditReader.php

```diff
@@ -424,12 +434,8 @@ public function findEntitiesChangedAtRevision($revision)
             }
 
             foreach ($classMetadata->associationMappings as $assoc) {
-                if (
-                    ($assoc['type'] & ClassMetadata::TO_ONE) > 0
-                    && true === $assoc['isOwningSide']
-                    && isset($assoc['targetToSourceKeyColumns'])
-                ) {
-                    foreach ($assoc['targetToSourceKeyColumns'] as $sourceCol) {
+                if (self::isToOneOwningSide($assoc)) {
+                    foreach (self::getTargetToSourceKeyColumns($assoc) as $sourceCol) {
```

Comment:
The `targetToSourceKeyColumns` field [is only available](https://github.com/search?q=repo%3Adoctrine%2Form+targetToSourceKeyColumns&type=code) in [`ToOneOwningSideMapping`](https://github.com/doctrine/orm/blob/1153b9468ccfef452625064c66b4a135d0ea8a46/src/Mapping/ToOneOwningSideMapping.php#L20)